README.md

```diff
@@ -86,22 +86,45 @@ Here is an example output:
 ```console
 $ sonar ps
 
-2023-07-27T18:29:17+02:00,somehost,8,user,,alacritty,3.7,214932
-2023-07-27T18:29:17+02:00,somehost,8,user,,slack,2.4,1328412
-2023-07-27T18:29:17+02:00,somehost,8,user,,X,0.8,173148
-2023-07-27T18:29:17+02:00,somehost,8,user,,brave,15.5,7085968
-2023-07-27T18:29:17+02:00,somehost,8,user,,.zoom,37.8,1722564
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=.vim-wrapped,cpu%=1.9,cpukib=7228,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=node,cpu%=1.8,cpukib=79332,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=slack,cpu%=0.7,cpukib=591720,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=X,cpu%=1.5,cpukib=224416,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=brave,cpu%=12.1,cpukib=3075300,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=alacritty,cpu%=1.2,cpukib=286196,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=sonar,cpu%=9,cpukib=372,gpus={},gpu%=0,gpumem%=0,gpukib=0
 ```
 
 The columns are:
-- time stamp
-- hostname
-- number of cores on this node
-- user
-- Slurm job ID (empty if not applicable)
-- process
-- CPU percentage (as they come out of `ps`)
-- memory used in KiB
+- `v`: version (to make data migrations easier in future)
+- `time`: local time stamp
+- `host`: host name
+- `cores`: number of cores on this node
+- `user` : username owning the process/command
+- `job`: job ID (0 if not applicable)
+- `cmd`: process/command
+- `cpu%`: CPU percentage (as they come out of `ps`)
```

Comment:
On a more general note I think we want more precision in this documentation.  Here's what I use elsewhere though I'd like to revisit some of these:

- version is on the format n.m.o
- time is ISO time without fractional seconds but with TZO
- host is FQDN
- cores is positive integer
- job id is positive integer; not convinced that zero should be legal here?
- user is not simply the user name, it can also be `_unknown_` and `_zombie_<something>`
- cpu% is percent of one core
- the card indices in gpus is 1-based (I'm not wedded to this; discuss)
- gpu% is sum across cards
- gpumem% is % of memory across all cards
- gpukib is sum across cards

Reply:
Reading the code some more I think gpu% and gpumem% and gpukib are sums across the cards that are used by the process whose information we're recording.  I'll look at this even more tomorrow.

Reply:
Good point about not allowing 0 job ID if it is not applicable. But then it should be positive integer or "none"? Or something like that?

Reply:
Having thought about this a little more, I guess "0" for "none" is better than a special token.  And "none" may be an unavoidable case in some scenarios.  On our systems without job queues, I run sonar with --batchless so that it synthesizes a nonzero job ID.  On systems with job queues, I guess "0" is something we use for processes that don't have job IDs in the queue.  I think I will be fine with that.  So consider my objection void.  (Unless you think there are batch queue systems that might use 0 for a valid job ID.)